### ISSUE TITLE
Add tooltip to text formatting buttons

### ISSUE THREAD

**Issue by isabela-pf** [jupytercalpoly/jupyterlab-richtext-mode#37]
The formatting buttons added through this extension are missing tooltips. This makes them inconsistent with other toolbar and buttons elsewhere in the interface, leads to poor user experience, and may impact accessibility.

It looks like JupyterLab currently labels tooltips with `title` (as is frequently recommended), but  this may be a [problem](https://github.com/google/material-design-icons/issues/872) with the current implementation of icons via the Material Icons font.
